src/hooks/init/init_doctor.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import type { Hook } from '@oclif/core';
+import { Logger } from '@salesforce/core';
+import { VersionCommand } from '@oclif/plugin-version';
+import { Doctor } from '../../doctor';
+
+const log = Logger.childFromRoot('plugin-plugin-info:init_doctor');
```

Comment:
nit: why such a long logger name?  `plugin-info` is useful but `plugin-plugin-info`?